components/cards/CardEvent.tsx

```diff
@@ -0,0 +1,172 @@
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import CardMedia from '@mui/material/CardMedia'
+import Typography from '@mui/material/Typography'
+
+import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import Box from '@mui/material/Box'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { useTheme } from '@mui/material/styles'
+
+
+type CardEventProps = {
+  title: string
+  date: string
+  time: { start: string; end: string }
+  location: string
+  description: string
+  buttonLink: string
+  imageSrc: string
+  imageAlt: string
+}
+
+const CardEvent = ({
+  title,
+  date,
+  time,
+  location,
+  description,
+  buttonLink,
+  imageSrc,
+  imageAlt
+}: CardEventProps) => {
+  const theme = useTheme()
+  const extraSmallScreen = useMediaQuery(theme.breakpoints.only('xs'))
+  const mediumScreen = useMediaQuery(theme.breakpoints.up('md'))
+  const largeScreen = useMediaQuery(theme.breakpoints.up('lg'))
+
+  if (extraSmallScreen || largeScreen) {
```

Comment:
I see your dilemma with the three breakpoints! At first glance it seems like it should be as simple as shifting the MUI Stack from 'row' to 'column', but that tablet breakpoint is tricky. Great use of the useMediaQuery utility. I used it on the home page too, because I couldn't figure out another way to change between <Typography /> variants!